docker-compose.yml

```diff
@@ -1,28 +1,28 @@
-web:
-  build: .
-  volumes:
-    - .:/champaign
-    - ../sou-assets:/sou-assets
-  ports:
-    - "3000:3000"
-  links:
-    - db
-    - redis
-  environment:
-    PG_USERNAME: postgres
-    PG_HOST: db
-  env_file:
-    - env.yml
+#web:
+#  build: .
+#  volumes:
+#    - .:/champaign
+#    - ../sou-assets:/sou-assets
+#  ports:
+#    - "3000:3000"
+#  links:
+#    - db
+#    - redis
+#  environment:
+#    PG_USERNAME: postgres
+#    PG_HOST: db
+#  env_file:
+#    - env.yml
```

Comment:
Why are you leaving this commented?

Reply:
I've commented it out because I'm for now not using the `web` container in my local setup any more. With the changes to the Dockerfile, it's no longer useful for development, so I'd need to make a separate development Dockerfile. I wanted to leave it in commented rather than removing it entirely since I might do that later at some point. The compose file with the web container commented out is still useful for local development because it runs redis and postgres for you and binds them to the same ports they'd run on on localhost.

Reply:
Makes sense, thanks for the explanation.